src/overlays/actors/ovl_En_Daiku2/z_en_daiku2.c

```diff
@@ -22,56 +53,525 @@ const ActorInit En_Daiku2_InitVars = {
     (ActorFunc)EnDaiku2_Draw,
 };
 
+static u16 sTextIDs[] = {
+    0x32CB, 0x32CC, 0x32CD, 0x32CE, 0x32CF, 0x32D0, 0x32D1, 0x32D2,
+};
 
-// static ColliderCylinderInit sCylinderInit = {
-static ColliderCylinderInit D_80BE7900 = {
-    { COLTYPE_NONE, AT_NONE, AC_NONE, OC1_ON | OC1_TYPE_ALL, OC2_TYPE_2, COLSHAPE_CYLINDER, },
-    { ELEMTYPE_UNK0, { 0x00000000, 0x00, 0x00 }, { 0xF7CFFFFF, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_NONE, OCELEM_ON, },
+static ColliderCylinderInit sCylinderInit = {
+    {
+        COLTYPE_NONE,
+        AT_NONE,
+        AC_NONE,
+        OC1_ON | OC1_TYPE_ALL,
+        OC2_TYPE_2,
+        COLSHAPE_CYLINDER,
+    },
+    {
+        ELEMTYPE_UNK0,
+        { 0x00000000, 0x00, 0x00 },
+        { 0xF7CFFFFF, 0x00, 0x00 },
+        TOUCH_NONE | TOUCH_SFX_NORMAL,
+        BUMP_NONE,
+        OCELEM_ON,
+    },
     { 20, 60, 0, { 0, 0, 0 } },
 };
 
+void func_80BE61D0(EnDaiku2* this) {
+    if ((this->unk_27A != -1) && (this->unk_258 != 0)) {
+        if (func_8013D68C(this->unk_258, this->unk_25C, &this->unk_268) == 0) {
+            Actor_MarkForDeath(&this->actor);
+        }
+    }
+}
+
+void EnDaiku2_Init(Actor* thisx, GlobalContext* globalCtx) {
+    EnDaiku2* this = THIS;
+    s32 day = gSaveContext.day;
+
+    this->actor.colChkInfo.mass = 0xFF;
+    ActorShape_Init(&this->actor.shape, 0.0f, func_800B3FC0, 40.0f);
+    SkelAnime_InitSV(globalCtx, &this->skelAnime, &D_0600A850, &D_06002FA0, this->limbDrawTable,
+                     this->transitionDrawTable, 17);
+    this->actor.targetMode = 0;
+    Collider_InitAndSetCylinder(globalCtx, &this->collider, &this->actor, &sCylinderInit);
+    this->unk_278 = this->actor.params & 0x7F;
+    this->unk_27A = (this->actor.params >> 7) & 0x3F;
```

Comment:
Something we've been doing in MM is using Macros for params like this. See https://github.com/zeldaret/mm/blob/ade921a2c00f8835c1e78b09a684d347f362fc1e/src/overlays/actors/ovl_En_Suttari/z_en_suttari.h#L11 as an example

Reply:
What would I call the macro if I don't know what it does?

Reply:
Good question. Kenix could probably answer when he passes by :p He's usually the one enforcing this. Although I can't remember if it's more of a later documentation step

Reply:
We are needing the macro, just give it an unknown name and stay consistent in the theme.